recipes/pseudo_dojo/meta.yaml

```diff
@@ -0,0 +1,64 @@
+{% set name = "pseudo_dojo" %}
+{% set version = "0.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/abinit/pseudo_dojo/archive/v{{ version }}.tar.gz
+  sha256: 8330a8b4dba96caca62be4f0fcd2fa869bdd103806630c2da9ee19932512ea10
+  patches:
+    - pymatgen.patch
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - six
+    - numpy
+    - scipy
+    - tabulate
+    - pandas
+    - jsonschema
+    - atomicfile
+    - periodic-table-plotter
+    - monty
+    - ipython
+    - nbformat
+    - matplotlib-base
+    - pymatgen
+    - abipy >=0.2.0
+
+test:
+  imports:
+    - pseudo_dojo
+
+about:
+  home: http://www.pseudo-dojo.org/
+  license: GPL-3.0-or-later
```

Comment:
The license file on github says GPL 2.1 or later. Are you sure about this one?

Reply:
(either case that's a bit odd since it's using abipy which is 2.0 only.)

Reply:
To my knowledge GPL 2.1 is not an official SPDX license, so I chose the next higher one. If you recommend to use `GPL-2.0-or-later` I am fine with that as well.

Reply:
What's GPL 2.1 actually? I only know of LGPL 2.1.

Reply:
Sorry, you are right. It is LGPL-2.1-or-later